packages/editor/src/components/rich-text/format-edit.js

```diff
@@ -43,7 +44,7 @@ function FillInserterListItem( props ) {
 					return null;
 				}
 
-				return <InserterListItem { ...props } />;
+				return <InserterListItem { ...props } icon={ normalizeIconObject( props.icon ) } />;
```

Comment:
Is the `normalizeIconObject` call still necessary here, shouldn't we call it on registration instead?

Reply:
We don't have access to `InserterListItem` unfortunately as it is part of the `edit` method.

Reply:
Not sure what's the relation with `InserterListItem`. I was just suggesting keeping this line `return <InserterListItem { ...props } />` here and calling `normalizeIconObject`inside `registerFormatType`

Reply:
Maybe, I'm not understanding this properly, but this icon is supposed to be the format icon right?

Reply:
Nope, it comes from the component directly

Reply:
Then, why do we even need to normalize it?

Reply:
Regardless, shouldn't we import `normalizeIconObject` from `rich-text` here?